types/webpack-assets-manifest/index.d.ts

```diff
@@ -4,11 +4,13 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.7
 
-import { Plugin } from "webpack";
+/// <reference types="node" />
+import { Compiler } from "webpack";
 import { SyncHook, SyncWaterfallHook } from "tapable";
 
-declare class WebpackAssetsManifest extends Plugin {
+declare class WebpackAssetsManifest {
```

Comment:
Any idea what arguments should be passed here? Webpack types allow 1-2, 3 are here though

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/52782/files#diff-0806b1c7645f696161960e912873b215e563758171ad7d3bdf414d3238daa1beR24